contracts/GitcoinAirdrop.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+
+contract GitcoinAirdrop is ERC721, Ownable {
+    using Strings for uint256;
+
+    uint256 public PLATFORM_FEE;
+    uint256 private _nextTokenId = 1;
+
+    struct Project {
+        string id;
+        string baseTokenURI;
+        uint256 tokenIdStart;
+        uint256 tokenIdEnd;
+    }
+
+    mapping(string => Project) public projects;
+    mapping(uint256 => string) public tokenToProjectId;
+    mapping(uint256 => address) public tokenToOwner;
+
+    event NFTsMinted(
+        string projectId,
+        uint256 tokenIdStart,
+        uint256 tokenIdEnd
+    );
+    event ProjectDeleted(string projectId);
+
+    constructor(
+        string memory _name,
+        string memory _symbol,
+        uint256 _platformFee
+    ) ERC721(_name, _symbol) Ownable(msg.sender) {
+        PLATFORM_FEE = _platformFee;
+    }
+
+    function mintProjectNFTs(
+        string calldata _projectId,
+        string calldata _baseTokenURI,
+        address[] calldata contributors
+    ) external payable {
+        require(msg.value >= PLATFORM_FEE, "Insufficient platform fee");
+        require(
+            bytes(projects[_projectId].id).length == 0,
+            "Project already exists"
+        );
+        require(contributors.length > 0, "No contributors provided");
+
+        uint256 tokenIdStart = _nextTokenId;
+        uint256 tokenIdEnd = tokenIdStart + contributors.length - 1;
+
+        projects[_projectId] = Project({
+            id: _projectId,
+            baseTokenURI: _baseTokenURI,
+            tokenIdStart: tokenIdStart,
+            tokenIdEnd: tokenIdEnd
+        });
+
+        for (uint256 i = 0; i < contributors.length; i++) {
+            uint256 tokenId = tokenIdStart + i;
+            _mint(contributors[i], tokenId);
+            tokenToOwner[tokenId] = contributors[i];
```

Comment:
Why do we need this? Standard NFT already has a track of address to token id isn't it?